### ISSUE TITLE
Please upgrade your code to use `jumpTo` warning

### ISSUE THREAD

**Issue by vlivanov** [react-navigation/react-navigation#4362]
Using createTabNavigator causes a warning with the following text: "Method `jumpToIndex` is deprecated. Please upgrade your code to use `jumpTo` instead. Change your code from `jumpToIndex(1)` to `jumpTo('Parties')."
However, the tab switching works fine.

This is caused by the following code in react-native-tab-view(0.7.7): 

```
console.warn(
      'Method `jumpToIndex` is deprecated. Please upgrade your code to use `jumpTo` instead.',
      `Change your code from \`jumpToIndex(${index})\` to \`jumpTo('${key}').\``
    );

```
![image](https://user-images.githubusercontent.com/1196106/40622517-b3fd5a2a-62aa-11e8-8804-b63750b66be4.png)

No warning should appear.

Basically any tabNavigator should generate this problem.

| software         | version
| ---------------- | -------
| react-navigation | 2.0.4
| react-native-tab-view | 0.7.7
| react-native     |  0.52.1
| node             |  8.9.1
| yarn      | 1.3.2

**Comment by davidzhang12138** [comment]
same 
"react": "16.3.1",
"react-native": "0.55.4",
"react-navigation": "^2.0.4",

**Comment by vlivanov** [comment]
Looks like the solution is to use react-navigation-tabs as shown [here](https://github.com/react-navigation/react-navigation/issues/4297):

import { createBottomTabNavigator, BottomTabBar } from 'react-navigation-tabs'

**Comment by clucasalcantara** [comment]
@vlivanov This isn't a solution, I've tried to replace TabNavigator to remove the warning, but for the result, I cant use a few properties that I have on the tabNavigator (as animation) available, so those sections with still have references to jumpToIndex won't be replaced?

**Comment by brentvatne** [comment]
we don't support those properties anymore, if you'd like to continue to use them you might want to create a fork of the deprecated tab navigators, remove the warnings, and so on
